docs/high/dataset.rst

```diff
@@ -614,3 +614,16 @@ Reference
     .. attribute:: parent
 
         :class:`Group` instance containing this dataset.
+
+    .. attribute:: __bool__
```

Comment:
`__bool__()` is a method, so we should use the `.. method::` directive.

Reply:
Ah yes, I also relocated the entries in the files accordingly.